docs/csharp/programming-guide/statements-expressions-operators/statements.md

```diff
@@ -21,10 +21,10 @@ The actions that a program takes are expressed in statements. Common actions inc
 |[Expression statements](expressions.md)|Expression statements that calculate a value must store the value in a variable. For more information, see [Expression Statements](#expression-statements).|  
-|[Selection statements](../../../csharp/language-reference/keywords/selection-statements.md)|Selection statements enable you to branch to different sections of code, depending on one or more specified conditions. For more information, see the following topics:<br /><br /> [if](../../../csharp/language-reference/keywords/if-else.md), [else](../../../csharp/language-reference/keywords/if-else.md), [switch](../../../csharp/language-reference/keywords/switch.md), [case](../../../csharp/language-reference/keywords/switch.md)|  
-|[Iteration statements](../../../csharp/language-reference/keywords/iteration-statements.md)|Iteration statements enable you to loop through collections like arrays, or perform the same set of statements repeatedly until a specified condition is met. For more information, see the following topics:<br /><br /> [do](../../../csharp/language-reference/keywords/do.md), [for](../../../csharp/language-reference/keywords/for.md), [foreach](../../../csharp/language-reference/keywords/foreach-in.md), [in](../../../csharp/language-reference/keywords/foreach-in.md), [while](../../../csharp/language-reference/keywords/while.md)|  
-|[Jump statements](../../../csharp/language-reference/keywords/jump-statements.md)|Jump statements transfer control to another section of code. For more information, see the following topics:<br /><br /> [break](../../../csharp/language-reference/keywords/break.md), [continue](../../../csharp/language-reference/keywords/continue.md), [default](../../../csharp/language-reference/keywords/switch.md), [goto](../../../csharp/language-reference/keywords/goto.md), [return](../../../csharp/language-reference/keywords/return.md), [yield](../../../csharp/language-reference/keywords/yield.md)|  
-|[Exception handling statements](../../../csharp/language-reference/keywords/exception-handling-statements.md)|Exception handling statements enable you to gracefully recover from exceptional conditions that occur at run time. For more information, see the following topics:<br /><br /> [throw](../../../csharp/language-reference/keywords/throw.md), [try-catch](../../../csharp/language-reference/keywords/try-catch.md), [try-finally](../../../csharp/language-reference/keywords/try-finally.md), [try-catch-finally](../../../csharp/language-reference/keywords/try-catch-finally.md)|  
+|Selection statements|Selection statements enable you to branch to different sections of code, depending on one or more specified conditions. For more information, see the following topics:<br /><br /> [if](../../../csharp/language-reference/keywords/if-else.md), [else](../../../csharp/language-reference/keywords/if-else.md), [switch](../../../csharp/language-reference/keywords/switch.md), [case](../../../csharp/language-reference/keywords/switch.md)|  
```

Comment:
Rather than a comma-separated series on one line, I'd think I would do the statements as bulleted list items on separate lines.

Reply:
That comment affects every row on this page. I create https://github.com/dotnet/docs/issues/12596 to address this in a separate PR.